### PR TITLE
Add class InverseDistanceWeightingInterpolation

### PR DESCRIPTION
**Pull request by nrelun**
…se distance weighting interpolation

### CODE REVIEW

File: lib/src/Base/Func/InverseDistanceWeightingInterpolation.cxx [hunk 1]
```diff
@@ -0,0 +1,262 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief An implementation from samples
+ *
+ *  Copyright 2005-2018 Airbus-EDF-IMACS-Phimeca
```
Comment (jschueller) [inline lib/src/Base/Func/InverseDistanceWeightingInterpolation.cxx]: 2019

**Comment by jschueller** [comment]
TODO: update the doc

**Comment by jschueller** [comment]
great! I have only minor comments

File: lib/src/Base/Func/InverseDistanceWeightingInterpolation.cxx (hunk 1 above)
Comment (nrelun) [reply lib/src/Base/Func/InverseDistanceWeightingInterpolation.cxx]: I am currently writing the doc

File: python/src/InverseDistanceWeightingInterpolation_doc.i.in [hunk 2]
```diff
@@ -0,0 +1,57 @@
+%feature("docstring") OT::PInverseDistanceWeightingInterpolation
```
Comment (nrelun) [inline python/src/InverseDistanceWeightingInterpolation_doc.i.in]: should be InverseWeightingDistanceInterpolation

**Comment by jschueller** [comment]
also, the t_InverseDistanceWeightingInterpolation.expout file is missing

**Comment by jschueller** [comment]
@nrelun If you dont plan on working on it anymore we could take it from here, sounds good ?

**Comment by jschueller** [comment]
wont be integrated, see Regis, sorry

**Pull request closed without merging**